bedrock/firefox/templates/firefox/mobile-2019.html

```diff
@@ -41,6 +41,11 @@ <h3>{{ _('Get automatic privacy on mobile') }}</h3>
           </div>
 
           <div class="mzp-c-hero-cta">
+            <div class="header-product-ctas">
```

Comment:
I think this needs `hide-from-legacy-ie` on it? Until we can make it a link to the get-app page.